R/tree_utils.R

```diff
@@ -10,7 +10,10 @@ generate_top_level_map <- function(pc) {
     tl_parent[i_not_assigned] <- pc$parent[match(tl_parent[i_not_assigned], pc$id)]
     i_not_assigned <- which(!tl_parent %in% tl_ids)
     if (length(i_not_assigned) >= prev_length) {
-      stop("Logical error: unassigned set did not shrink. Check file syntax and please report as a lintr bug.") # nocov
+      stop(
+        "Logical error: unassigned set did not shrink. Check file syntax and please report as a lintr bug.",
+        call. = FALSE
+      ) # nocov
```

Comment:
can you confirm this doesn't throw off code coverage? it looks like nocov will now apply to one line instead of 3.